estate/__manifest__.py

```diff
@@ -0,0 +1,19 @@
+{
+    'name': "Real Estate: 100% Stonks",
+    'version': '1.0',
+    'depends': ['base', 'hr'],
```

Comment:
`base` is a dependency of `hr`, no need to have them both here
```suggestion
    'depends': ['hr'],
```